src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -7,45 +7,85 @@ import {
   LARGE_SCREEN_MEDIA_QUERY,
   MEDIUM_SCREEN_MEDIA_QUERY,
 } from 'utils/mediaQueries'
+import { Button } from 'components/buttons'
 import { quoteDetailsHasAddress } from '../utils'
 import { getAddress } from '../Checkout/InsuranceSummaryDetails'
+import { DetailsModal } from './DetailsModal'
 
 type Props = {
   offerData: OfferData
+  refetchOfferData: () => Promise<void>
 }
 
 const Wrapper = styled.div`
-  padding: 2.5rem 0 1rem;
-  color: ${colorsV3.white};
+  padding: 3.5rem 0 0;
+  color: ${colorsV3.gray900};
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
     padding-top: 0;
+    color: ${colorsV3.white};
   }
 `
 const Headline = styled.h1`
-  margin: 0;
+  margin-bottom: 0.5rem;
   text-transform: uppercase;
-  font-size: 1rem;
+  font-size: 0.875rem;
+  line-height: 1.375rem;
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    font-size: 1rem;
+    line-height: 1.5rem;
+  }
 `
 const OfferInfoWrapper = styled.div`
-  padding-bottom: 2rem;
+  padding-bottom: 1.5rem;
 `
 const NameAndCoInsured = styled.div`
-  font-size: 2rem;
+  font-size: 1.5rem;
+  line-height: 2rem;
+  display: inline-block;
+  margin-right: 0.5rem;
 
   ${MEDIUM_SCREEN_MEDIA_QUERY} {
     font-size: 3rem;
+    line-height: 3.5rem;
+  }
+`
+
+const Separator = styled.span`
+  display: inline;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    display: none;
```

Comment:
I think it makes sense to use pseudo elements for these type of things rather than adding it in the markup :) 
```suggestion
const NameAndCoInsured = styled.div`
  ...
  &:after {
      content: ',';
      display: inline;
  
      ${MEDIUM_SCREEN_MEDIA_QUERY} {
        display: none;
      }
    }
```

Reply:
Nice solution!